play/type-shooter/js/app.js

```diff
@@ -158,9 +230,13 @@ function endGame() {
     document.getElementById("playerScore").innerHTML = playerScore;
 
     // calculate timeSurvived and display it in the form 00:00
-    let timeSurvived = ((timer / 60).toFixed() < 10 ? "0" + (timer / 60).toFixed() : (timer / 60).toFixed());
+    let timeSurvived = (Math.floor(timer / 60) < 10 ? "0" + Math.floor(timer / 60) : Math.floor(timer / 60));
     timeSurvived += ":" + (timer % 60 < 10 ? "0" + timer % 60 : timer % 60);
-    document.getElementById("timeSurvived").innerHTML = timeSurvived;
+
+    let timeStats = document.querySelectorAll("#timeSurvived");
+    timeStats.forEach(timeStat => {
```

Comment:
With arrow functions with only one statement, you can make them compact where you get rid of the brackets and semicolon to make it fit all in one line. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions